src/collective/nitf/upgrades/v2002/configure.zcml

```diff
@@ -0,0 +1,19 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    i18n_domain="collective.nitf">
+
+  <genericsetup:upgradeSteps
+      source="2001"
+      destination="2002"
+      profile="collective.nitf:default">
+
+    <genericsetup:upgradeStep
+        title="Remove NITF portlet"
+        description="Walk over the site to search and remove portlet assignments."
+        handler=".remove_nitf_portlet"
+        />
```

Comment:
This upgrade step will have to be removed when the BBB is removed. Maybe this should be documented here.